src/test/java/de/tum/in/www1/artemis/exam/ExamUtilService.java

```diff
@@ -521,6 +531,37 @@ public Channel addExamChannel(Exam exam, String channelName) {
         return conversationRepository.save(channel);
     }
 
+    /**
+     * Creates and saves a Post for the given Channel.
+     *
+     * @param channel The Channel for which the Post should be created
+     * @param title   The title of the Post
+     * @param author  The author of the Post
+     * @return The newly created Post
+     */
+    public Post createPost(Channel channel, String title, User author) {
+        Post post = new Post();
+        post.setTitle(title);
+        post.setConversation(channel);
+        post.setAuthor(author);
+        return postRepository.save(post);
+    }
+
+    /**
+     * Creates and saves an AnswerPost for the given Post.
+     *
+     * @param post    The Post for which the AnswerPost should be created
+     * @param content The content of the AnswerPost
+     * @param author  The author of the AnswerPost
+     */
+    public void createAnswerPost(Post post, String content, User author) {
+        AnswerPost answerPost = new AnswerPost();
+        answerPost.setContent(content);
+        answerPost.setPost(post);
+        answerPost.setAuthor(author);
+        answerPostRepository.save(answerPost);
+    }
```

Comment:
Is this intentionally included in this PR? I do not see the connection right away 😅